pkg/api/types.go

```diff
@@ -403,6 +404,13 @@ const (
 	PodUnknown PodPhase = "Unknown"
 )
 
+type PodCondition string
+
+// These are the valid service statuses of pods.
+const (
+	PodInService = "InService"
```

Comment:
I would like to report Liveness status, also, and probably other conditions later.